### PR TITLE
Guest reservation

### PR DESCRIPTION
**Pull request by ngdo-pro**
Hey guys, 

Here is the guest reservation,

Please, remember to : 

+ app/console doctrine:migrations:migrate
+ app/console doctrine:fixture:load

To test it : 

+ Log as john@email.com
+ Go to http://localhost:8000/bapteme-de-chef/reserver/ (id of a baptismHasUser you have in your database)
+ Test and check your database to verify that your reservation is registered

### CODE REVIEW

**Review by shog189 — APPROVED** [review]
:+1:

**Review by dbeauseigneur — APPROVED** [review]
OK

File: src/AppBundle/Form/BaptismGuestType.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('guestCount', ChoiceType::class, array(
+                'choices' => array(
+                    '1' => 1,
+                    '2' => 2,
+                    '3' => 3,
+                    '4' => 4
```
Comment (Kaji-U) [inline src/AppBundle/Form/BaptismGuestType.php:24]: Here we see four choices, finally, we fix the maximum guests invitations per user ? Just a question

**Review by Kaji-U — APPROVED** [review]
It's ok for me

**Review by wafaworks — COMMENTED** [review]
👍

**Pull request merged**